modules/@apostrophecms/widget-type/ui/apos/components/AposWidgetEditor.vue

```diff
@@ -177,7 +189,12 @@ export default {
     getDefault() {
       const widget = {};
       this.schema.forEach(field => {
-        widget[field.name] = field.def ? klona(field.def) : field.def;
+        // Do not simply check if `field.def` is truthy, this in not enough
+        // since a def as an empty string must be considered valid:
+        const hasDefaultValue = Object.prototype.hasOwnProperty.call(field, 'def');
```

Comment:
hasOwn, no need for prototype etc.

Reply:
I was wondering, having not found its usage in the code, I chose `hasOwnProperty`.
Good thing that we can use `Object.hasOwn` now